index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="/">Tolu's Logo</a></li>
+                <li><a href="/"><i class="fas fa-bars hamburger-icon menu"></i></a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <div class="main-section">
+            <article class="headline">
+                <p>Hey There.<br> I am Tolu</p>
+                <h4>I am a Software Developer</h4>
+                <h6>I can help you build a product, feature or website. Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, do not hesitate to contact me.</h6>
+            </article>
+
+            <div class="social-icons">
+                <ul>
+                    <li><a href="/"><i class="fab fa-github"></i></a></li>
+                    <li><a href="/"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a href="/"><i class="fab fa-angellist"></i></a></li>
+                    <li><a href="/"><i class="fab fa-twitter"></i></a></li>
+                    <li><a href="/"><i class="fab fa-medium-m"></i></a></li>
+                </ul>
+            </div>
+
+            <div class="group-image">
+                <img src="/image/Group.png" alt="vector-grouped-image">
+            </div>
```

Comment:
- [ ] I think it will be better if you set the background image using CSS instead of HTML. Because the image is for decoration purposes and not related to the page content.

Reply:
Yes, I agree. I have done that. Thank you.